packages/bridge/src/vl/time-series/TimeSeries.ts

```diff
@@ -116,6 +143,23 @@ export class TimeSeries {
       this._animation.pause();
     });
   }
+
+  private _createAnimation() {
+    const s = this._carto.expressions;
+
+    return s.animation(
```

Comment:
I see your point here on creating something more easily from the string representation

Reply:
Yep, VL String API is :boom: